### ISSUE TITLE
Quartz Guide

### ISSUE THREAD

**Issue by McCartneyAC** [datajtext/DataJournalismTextbook#8]
you may consider bringing in this guide for data journalists as well: 
https://github.com/Quartz/bad-data-guide

**Comment by mattwaite** [comment]
It's very much in the plans. The question is how. Do we literally bring it in wholesale or do we link out to it? Do we weave parts of it into different modules or make an entire module on bad data? Or is the answer yes to all of this? I don't see a world where we don't make use of their bad data guide, but how isn't clear yet.

**Comment by amandabee** [comment]
I feel like a round up of newsroom standards makes sense as a free standing module. 

Style guides and extensions of style guides.

**Comment by sarahcnyt** [comment]
@McCartneyAC :  I agree with Matt that it will definitely be used in the project in some way. We have some short pieces in the works under the broad heading of Lesson Zero -- things like data file formats,  tidy data, basic programming like loops, etc.  I'm linking to the Quartz guide under tidy data (in progress). I expect we'll also have a whole module on cleaning data.  

@amandabee  good idea on newsroom standards. Take a look at the data diary under replication as one part of that, though I get it that there should be more. (Also, I forgot to link to your github repos in the updated readme)
